backend/sirius-components-diagrams/src/test/java/org/eclipse/sirius/components/diagrams/renderer/UnsynchronizedDiagramTests.java

```diff
@@ -154,7 +156,20 @@ public void testUnsynchronizedRenderingOnRootAfterDeletion() {
      */
     @Test
     public void testUnsynchronizedRenderingOnNodeContainer() {
-        DiagramDescription diagramDescription = this.getDiagramDescription();
+
+        Function<VariableManager, List<?>> variableTester = new Function<>() {
```

Comment:
what is a `variableTester`? It does not seems to test anything, isn't that a `semanticElementsProvider`?